From 276fbb806b70d79096f2deb5c42d36382b3ecf62 Mon Sep 17 00:00:00 2001
From: Jason Kridner <jdk@ti.com>
Date: Wed, 18 Sep 2019 19:41:04 +0000
Subject: [PATCH] Hack to disable excessive warnings common with many cameras

Prevents re-occuring messages from OpenCV like:
Corrupt JPEG data: 7 extraneous bytes before marker 0xd1
---
 jdmarker.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/jdmarker.c b/jdmarker.c
index c9c7ef639..efcea7986 100644
--- a/jdmarker.c
+++ b/jdmarker.c
@@ -923,7 +923,7 @@ next_marker(j_decompress_ptr cinfo)
   }
 
   if (cinfo->marker->discarded_bytes != 0) {
-    WARNMS2(cinfo, JWRN_EXTRANEOUS_DATA, cinfo->marker->discarded_bytes, c);
+    //WARNMS2(cinfo, JWRN_EXTRANEOUS_DATA, cinfo->marker->discarded_bytes, c);
     cinfo->marker->discarded_bytes = 0;
   }
 
